template/experiment/index.js

```diff
@@ -0,0 +1,37 @@
+// import React from 'react';
```

Comment:
Remove the code that is currently commented out. If the code will be uncommented later, consider adding text to indicate that it will be restored in the future (something like `// TOOD (Restore in future):` .

Reply:
Done - currently only dev mode is supported. Will remove once the remainder has been implemented without React components